README.md

```diff
@@ -61,11 +63,12 @@ const foo = {|
 <details><summary>Click here to see the desugared version</summary>
 
 ```js
-const foo = Object.seal({
-  a: Object.seal({
-    b: Object.seal({
-      c: Object.seal({
-        d: Object.seal({
+const seal = obj => Object.seal(Object.assign(Object.create(null), obj)));
+const foo = seal({
```

Comment:
For now let's inline the `seal` function as I want the examples to be mostly illustrative, without introducing theoretical helper methods. Otherwise I'm happy to merge this

Reply:
(Same for `freeze` too!)

Reply:
I tried it inine and it looks really ugly

Reply:
Maybe something like this?
```js
const foo = Object.seal({
  __proto__: null,
  a: Object.seal({
    __proto__: null,
    b: /* ... */
```

Reply:
oh yeah that's just way better, althought __proto__ is discouraged.